### ISSUE TITLE
app crashes at "send signature"

### ISSUE THREAD

**Issue by brenzi** [encointer/encointer-app#7]
tested with two devices (occurs on both)
* Samsung G800F, Android 6.0.1, API 23
* Wiko Jerry, Android 6.0, API 23
* SM J320F, Android 5.1.1, API 22

reproduce
* enter name and start
* enter 3 participants (although there are only 2)
* wait until they found each other
* tap "send signature"

```
09-09 18:04:41.339 10207-10207/com.encointer.signer E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.encointer.signer, PID: 10207
    java.lang.NoClassDefFoundError: Failed resolution of: Ljava/time/LocalDateTime;
        at com.encointer.signer.DeviceList.sendSignature(DeviceList.java:458)
        at com.encointer.signer.DevicesRecyclerViewAdapter$1.onClick(DevicesRecyclerViewAdapter.java:71)
        at android.view.View.performClick(View.java:5721)
        at android.widget.TextView.performClick(TextView.java:10955)
        at android.view.View$PerformClick.run(View.java:22620)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:7331)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "java.time.LocalDateTime" on path: DexPathList[[zip file "/data/app/com.encointer.signer-1/base.apk"],nativeLibraryDirectories=[/data/app/com.encointer.signer-1/lib/arm, /data/app/com.encointer.signer-1/base.apk!/lib/armeabi-v7a, /vendor/lib, /system/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
        at com.encointer.signer.DeviceList.sendSignature(DeviceList.java:458) 
        at com.encointer.signer.DevicesRecyclerViewAdapter$1.onClick(DevicesRecyclerViewAdapter.java:71) 
        at android.view.View.performClick(View.java:5721) 
        at android.widget.TextView.performClick(TextView.java:10955) 
        at android.view.View$PerformClick.run(View.java:22620) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:7331) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120) 
    	Suppressed: java.lang.ClassNotFoundException: java.time.LocalDateTime
        at java.lang.Class.classForName(Native Method)
        at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
        at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
        		... 13 more
     Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack trace available
```

**Comment by brenzi** [comment]
the function `LocalDateTime` is only supported with API>=26, so not with android 6.

**Comment by brenzi** [comment]
fixed with backport in 6c1f0ca
https://stackoverflow.com/questions/38922754/how-to-use-threetenabp-in-android-project
